Content.Client/Administration/UI/Notes/NoteEdit.xaml.cs

```diff
@@ -172,8 +186,9 @@ private void UpdatePermanentCheckboxFields()
     {
         ExpiryLabel.Visible = !PermanentCheckBox.Pressed;
         ExpiryLineEdit.Visible = !PermanentCheckBox.Pressed;
+        ExpiryLengthDropdown.Visible = !PermanentCheckBox.Pressed;
 
-        ExpiryLineEdit.Text = !PermanentCheckBox.Pressed ? DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss") : string.Empty;
+        ExpiryLineEdit.Text = !PermanentCheckBox.Pressed ? 7.ToString() : string.Empty;
```

Comment:
Since they can already select a week in the dropdown it might make sense to make the default 1 instead of 7.
```suggestion
        ExpiryLineEdit.Text = !PermanentCheckBox.Pressed ? 1.ToString() : string.Empty;
```

Reply:
My reasoning was that days would be the most versatile and common setting, while a week would be the most common expiration time! If you do 1 week instead, then if you only want to do like 1 day or something you'd have to click the button. I felt like having it be 7 days allows for a little more flexibility.

I have zero idea how this would turn out in reality though so if you think that would be better I'll change it!

Reply:
I would have argued the other way around, you might want to keep the note for 1 week, day or hour. Using the dropdown button is easier to change than having to delete the 7 from the text field and putting in a new number.

Reply:
Fair enough! I'll change it 🫡